build/Makefile

```diff
@@ -1,5 +1,6 @@
 export PACKER_CACHE_DIR := .cache
 export PACKER_VERSION := 1.9.4
+export PMM_SERVER_IMAGE ?= docker.io/perconalab/pmm-server-fb:PR-3664-e536f77
```

Comment:
Is the image name a left over? Or we want to keep it until FB is tested?

Reply:
Yes, should be replaced with `perconalab/pmm-server:3-dev-latest` before merge